scripts/shutdown_nodes.sh

```diff
@@ -0,0 +1,42 @@
+#/bin/sh
+
+NODES=$1
+if [ -z $NODES ]
+then
+    NODES=1
+fi
+
+echo "**** Number of nodes to be shutdown: $NODES ****"
+
+echo "------- Running unsafe reset all ---------"
```

Comment:
This section should be executed after systemd process has been stopped. Change the order of execution for this.